src/realitycheck/realitycheck.js

```diff
@@ -98,6 +96,11 @@ define(["jquery", "windows/windows", "websockets/binary_websockets", "lodash", '
 
                                 require(['text!realitycheck/realitycheck.html', "css!realitycheck/realitycheck.css"], function(html) {
                                     var div = $(html).i18n();
+                                    // Is there a better option?
+                                    if(win){
+                                        resolve();
+                                        return;
+                                    }
                                     win = windows.createBlankWindow($('<div/>'), {
```

Comment:
@arnabk The problem here's we're calling the init() twice. The second call is so close to previous one that it gets through the `if(win)` check to next statement. The best way I found to fix the problem was to move the check inside the promise. Do you know a better approach to this?